marshmallow_recipe/bake.py

```diff
@@ -102,7 +103,7 @@ def bake_schema(
 
 def get_field_for(
     type: Type[_T],
-    metadata: Mapping[str, Any],
+    metadata: collections.abc.Mapping[str, Any],
```

Comment:
would be great to use https://github.com/asottile/pyupgrade it will replace obsolete python automatically

Reply:
OK